fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/navigation/TabBar.kt

```diff
@@ -0,0 +1,85 @@
+package com.microsoft.fluentui.tokenized.navigation
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.compositionLocalOf
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.ControlTokens
+import com.microsoft.fluentui.theme.token.controlTokens.*
+import com.microsoft.fluentui.tokenized.tabItem.TabItem
+
+
+data class TabData(
+    var title: String,
+    var icon: ImageVector,
+    var selectedIcon: ImageVector = icon,
+    var selected: Boolean = false,
+    var onClick: () -> Unit,
+    var badge: @Composable (() -> Unit)? = null
+)
+
+val LocalTabBarInfo = compositionLocalOf { TabBarInfo() }
+val LocalTabBarToken = compositionLocalOf { TabBarTokens() }
+
+@Composable
+fun getTabBarInfo(): TabBarInfo {
+    return LocalTabBarInfo.current
+}
+
+@Composable
+fun getTabBarToken(): TabBarTokens {
+    return LocalTabBarToken.current
+}
+
+@Composable
+fun TabBar(
+    tabDataList: List<TabData>,
+    modifier: Modifier = Modifier,
+    selectedIndex: Int = 0,
+    tabTextAlignment: TabTextAlignment = TabTextAlignment.VERTICAL,
+    tabItemTokens: TabBarTabItemsTokens? = null,
```

Comment:
Lets use TabItemToken here itself since that is the component. Instead of creating New tokens, you can have an object with overridden values as the default value for this. No need to create new token

Reply:
removed TabItemToken